docs-source/content/userguide/managing-domains/model-in-image/usage.md

```diff
@@ -0,0 +1,155 @@
++++
+title = "Usage"
+date = 2020-03-11T16:45:16-05:00
+weight = 20
+pre = "<b> </b>"
+description = "Steps for creating and deploying Model in Image images and their associated domain resources."
++++
+
+
+#### Contents
+
+   - [WebLogic operator](#1-weblogic-operator)
+   - [WebLogic image](#2-weblogic-image)
+   - [Optional WDT model config map](#3-optional-wdt-model-config-map)
+   - [Required runtime encryption secret](#4-required-runtime-encryption-secret)
+   - [Secrets for model macros](#5-secrets-for-model-macros)
+   - [Domain resource attributes](#6-domain-resource-attributes)
+   - [Prerequisites for JRF domain types](#7-prerequisites-for-jrf-domain-types)
+
+#### Requirements
+
+Here's what's needed to create and deploy a typical Model in Image domain (though enumerated, you need not supply them in order):
```

Comment:
The parenthetical is a bit confusing.  It might be clearer to just have a second sentence saying, "These items do not need to be created in order."

Reply:
ok, less fancy but more clear. Done.